CONTRIBUTING.md

```diff
@@ -33,13 +33,40 @@ yarn build
 yarn build:watch
 ```
 
-## Run tests
+## Updating and adding packages
+
+Since this is a monorepo with several subrepos, if you want to update/add packages in the subrepos like `create-snowpack-app/app-template-11ty` you'll want to run the commands in the target subdirectory like
+
+```bash
+cd create-snowpack-app/app-template-vue
+yarn add vue@latest
+```
+
+## Tests
+
+Our test suite uses a mixture of unit, integration, and snapshot tests. We recommend running these before you submit a PR.
+
+### Running tests
+
+To run tests use:
```

Comment:
Should this also state to change directory to the top level / root first?

Reply:
How about 

```suggestion
From the repository's root folder, run
```

Reply:
True that thanks!